src/Mirage-Carousel/SystemWindow.extension.st

```diff
@@ -39,7 +39,7 @@ SystemWindow >> mirageThumbnailPropertySymbol [
 { #category : #'*Mirage-Carousel' }
 SystemWindow >> scheduleThumbnailUpdate [
 	([ 
-		MICarouselSettings thumbnailUpdateTime wait.
+		MICarouselSettings thumbnailUpdateTime seconds wait.
```

Comment:
To put it back here?

Reply:
This is a consequence of what I describe above.